### DISCUSSION TITLE
Fullscreen player

### DISCUSSION THREAD

**Discussion by SeliaLong** [Blinue/Magpie#807] [General]
im trying to make it work with old lowres anime, but its still leaves black bars around image. When i use older magpie verion 0.9.1 its works well and automaticly streches image to full screen. What could be an issue and how can i fix it?

**Comment by Blinue** [comment]
Please go to the scaling configuration page and add a suitable magnification algorithm, such as Lanczos, to the end of the scaling mode you are using.

**Reply by SeliaLong** [reply]
Thanks alot, that helped! 
One more thing i wanted to ask, are you planning to add FSR 3, or maybe even AMD frame generation to magpie? 
They both opensource and it would be awesome if you can do it!

**Reply by Blinue** [reply]
#802
